python/kserve/kserve/protocol/dataplane.py

```diff
@@ -23,6 +23,7 @@
 from ray.serve.handle import RayServeHandle, RayServeSyncHandle, DeploymentHandle
 
 from .rest.v2_datamodels import GenerateRequest, GenerateResponse
+from .rest.openai_datamodels import ChatCompletionRequest, ChatCompletionResponse
```

Comment:
This import file doesn't exists `openai_datamodels`. PR not updated yet?

Reply:
The imports should be synced now.